about.html

```diff
@@ -0,0 +1,118 @@
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+          </ul>
+        </nav>
+        <nav class="nav-bar">
+          <div class="ham">
+            <div class="line"></div>
+            <div class="line"></div>
+            <div class="line"></div>
+          </div>
+          <a class="ham-desk" href="index.html">
+            <img src="images/photos/cc_image.png" alt="Free thinking image and logo" class="cc_image">
+          </a>
+          <ul class="nav-desk-right">
+            <li class="nav-desk about"><a href="about.html">About</a></li>
+            <li class="nav-desk">Program</li>
+            <li class="nav-desk">Join</li>
+            <li class="nav-desk">Sponsor</li>
+            <li class="nav-desk">News</li>
+            <li class="nav-desk">CC Campaign</li>
+          </ul>
+        </nav>
+        <div class="about" id="about">
+          <section class="hero-1">
+            <div class="space"></div>
+            <div class="hello-0">"Hello, Free Thinkers!"</div>
+            <h2 class="title-0">FREE THINKING SUMMIT GLOBAL 2020</h2>
+            <div class="horizontal-rule"></div>
+            <p class="description-3">The FREE THINKING SUMMIT brings together the community of experts, academics, and
+              activists who comprise the Creative Commons affiliate network in a different country every two. This year,
+              we
+              hope to expand our invitation list including organizations and invdividuals</p>
+            <footer class="please">
+              Please contact us per Email for any further questions about FREE THINKING SUMMIT GLOBAL
+              2020!<br><strong><u>ftsummit2020@ftkorea.org</u></strong>
+            </footer>
+          </section>
+          <section class="creative">
+            <div class="title-1">FREE THINKING SUMMIT GLOBAL 2020 Logo</div>
+            <div class="horizontal-rule"></div>
+            <h4 class="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Ab accusamus, nobis eaque harum
+              quidem
+              sunt fugiat. Soluta, omnis facilis?</h4>
+            <img src="images/photos/cc_image.png" alt="Free thinking image and logo" class="cc">
+          </section>
+          <section class="past">
+            <header class="title-2">See the past FT Global Summits</header>
+            <div class="horizontal-rule"></div>
+            <h4 class="description-3">Take a look at the last two FT Global Summits which took place in Buenos Aires and
+              in
+              Warsaw</h4>
+            <ul class="past-one-and-two">
+              <li class="past-one">
+                <div class="year">2018</div>
+                <div class="text">CC Global Summit in Buenos Aires</div>
+              </li>
+              <li class="past-one">
+                <div class="year">2016</div>
+                <div class="text">CC Global Summit in Warsaw</div>
+              </li>
```

Comment:
- [ OPTIONAL ] I think it would be great if you made the images with a red background like the given design so kindly make it the same 💯🔥✅

- [ OPTIONAL ] I think it would be great if you decrease the width as shown in the given design 💪🏻💪🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/226077555-0639e15e-0c6a-4cd9-93d8-b019be2259e4.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/226077572-19758d8e-99cb-49a6-b041-111944e21ee9.png)

Reply:
Hi dear reviewer @Reem-lab, I was unable to change only the background image color without affecting the content of the whole container, like the text. Could you please show me how to do that? Thank you.

Reply:
Hello @danifromecuador I am not sure how to do it exactly but you can check this link maybe it helps https://youtu.be/Ni49a8gZQxY